### ISSUE TITLE
Consumable Inventory

### ISSUE THREAD

**Issue by atrium-complex** [grokability/snipe-it#8976]
- Snipe-IT Version: v5.0.11 build 5695 (gfd4ee6027)
 - OS: Ubuntu: 20.04
 - Web Server: Apache2
 - PHP Version: 7.2.34-8+ubuntu20.04.1+deb.sury.org+1

I would like to suggest the ability to assign consumable items to assets as well.

Use case:
My company does not track toner and paper to the user, we wish to track by the printer.

Assigning consumables directly to the printer allows for historical tracking of used toner/paper to the printer, allowing us preemptively issue toner based on dates and usage.

**Comment by cryogenx** [comment]
Good call, i would agree with this.

**Comment by tsdigital** [comment]
There are several feature requests for this, some dating back for 5 years.

**Comment by cryogenx** [comment]
@tsdigital We gave up and wrote our own inventory/IAM system..    could wait for this to happen

**Comment by snipe** [comment]
Cool. I’d love to hear how that works out. (I hope it works out really well. No bullshit) 

PRs are thing tho.

Glad you’ve figure out what works for you.

**Comment by tsdigital** [comment]
Wait, so this was closed because a single user said they no longer need it? It was linked to 5 other requests that were exactly the same.

**Comment by snipe** [comment]
Have you tried not being a dick on FOSS repos?

**Comment by tsdigital** [comment]
> Have you tried not being a dick on FOSS repos?

There was no being a dick here. Merely a simple question.

**Comment by snipe** [comment]
Fair. But this was not failure on our part. We have to work within the constraints we’re given.

**Comment by snipe** [comment]
If you have a different issue, open a new ticket

**Comment by tsdigital** [comment]
I understand development constraints. I just wanted to make sure this feature request wasn't getting closed prematurely. Based on my Googling it's probably the single most requested feature. 

I understand it may take some time.

**Comment by cryogenx** [comment]
> Cool. I’d love to hear how that works out. (I hope it works out really well. No bullshit) 
> 
> 
> 
> PRs are thing tho.
> 
> 
> 
> Glad you’ve figure out what works for you. 

Sorry wasn't trying to be rude or anything you have a great product that met almost every need we had, we just needed the ability to restock the inventory. 

You gave us a lot
If inspiration so we thank you for that. 

We also didn't have anyone on our team that knew the framework that snipe is written in. So with the time constraints we had to deal with we felt it prudent to buckle down for 2 weeks and just create it in the framework we used for our other projects.

I have and will continue to recommend snipe for a free AM tool.
